docker/Dockerfile

```diff
@@ -33,6 +33,8 @@ ENV CC=${CC}
 ENV CXX=${CXX}
 # Optional build arguments
 
```

Comment:
Do we still need those `ARCH` things? I would remove them if no.

Reply:
If our CMake accepts and uses this value I would like to have an ability to pass it in docker. Just in case.

Reply:
But you removed them, now it's only created through our build scripts and the name is IMO too generic.

Reply:
After discussion we will leave it as it is as it is used in other places as well.